### ISSUE TITLE
1.5.2 doesn't have "Routing assests" settings.

### ISSUE THREAD

**Issue by BusterN** [hiddify/hiddify-next#1030]
### Feature description

1.5.2 version doesn't have "Routing assets" feature in settings, so all traffic? including domestic? goes through VPN.

**Comment by Economic-PunPun** [comment]
I think they've removed "routing assets" Button to keep the UI minimal and user friendly.
But Routing assets are there and selected "Region" is excluded from tunneling.

**Comment by BusterN** [comment]
I checked it. Unfortunately all traffic goes through VPN now. Maybe reason is something else but it is true.

**Comment by wao1201** [comment]
> I checked it. Unfortunately all traffic goes through VPN now. Maybe reason is something else but it is true.

I rolled back to v1.4.0

**Comment by lm1618** [comment]
+1

**Comment by FizzVPN** [comment]
This issue still persists in 1.6.0. 
On 1.4.0 routing works well tho

**Comment by somebodywashere** [comment]
Regional routing in 1.5.2 and 1.6.0 doesn't work anymore. 
Only tested on Russia region tho.

**Comment by FlashWins** [comment]
yeah, it doesn't work correctly in Iran too. I reported it in this topic: https://github.com/hiddify/hiddify-next/issues/1065 . I think developer used v2ray rules instead of sing-box rules and it caused these problems.

**Comment by FlashWins** [comment]
@hiddify-com may we ask you to implement Sing-box files *.db and Sing-box rules instead of v2ray *.dat files & rules? the app worked greatly with those rules.

**Comment by BusterN** [comment]
Finally, I turned on the "Resolve destination" setting and it seems like worked for me. It's turned off by default.
